eslint-local-rules.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+/**
+ * This file contains any locally defined ESLint rules. They are picked up by
+ * eslint-plugin-local-rules and exposed as 'local-rules/<rule-name>'.
+ * See packages/@n8n_io/eslint-config/ for details.
+ *
+ * These are looked up in a directory above node_modules/, which means that for an
+ * npm workspace it needs to be located at the very root.
```

Comment:
I was hoping to be able to contain these within `packages/@n8n_io/eslint-config`, but the plugin only looks up its rules from above `node_modules` where it's loaded from, which currently has this sit at the top level.

Reply:
Oh wow, appreciate the trouble 🙏🏻 

On a side note, we should probably start publishing helpers such as this from within the n8n-io org namespace, what do you think?